kratos/includes/variables.h

```diff
@@ -89,6 +89,7 @@ namespace Kratos
     KRATOS_DEFINE_VARIABLE( double, RESIDUAL_NORM )
     KRATOS_DEFINE_VARIABLE( double, CONVERGENCE_RATIO )
 
+    KRATOS_DEFINE_VARIABLE( double, REFERENCE_TEMPERATURE )
```

Comment:
no registration and exposal?

Reply:
This variable is defined and registre already in mat_variables, should we keep this here?